### PR TITLE
Update checkout-payment-failed-emails.md

### PR DESCRIPTION
**Pull request by marklehm**
## Purpose of this pull request

This pull request (PR) corrects the information: Email is not sent to the customer but to an Admin user.

## Magento release version

- [x ] 2.4.x

   Specify a patch release number, if applicable:

- [x] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/sales/checkout-payment-failed-emails.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/sales/checkout-payment-failed-emails.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: Payment Failed Email
 group: marketing
 ---
```
Comment (leslietilling) [inline src/sales/checkout-payment-failed-emails.md:5]: I need some clarification for "an admin" -- Is it a Magento Admin user, and is the email configured somewhere?

File: src/sales/checkout-payment-failed-emails.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ title: Payment Failed Email
...
 
-A notification is sent to the customer if the payment method that is chosen during checkout fails to complete the transaction.
+A notification is sent to an admin if the payment method that is chosen during checkout fails to complete the transaction.
```
Comment (leslietilling) [inline src/sales/checkout-payment-failed-emails.md:6]:
```suggestion
A notification is sent to the store contact or another designated Admin user if the payment method that is chosen during checkout fails to complete the transaction.
```

**Comment by leslietilling** [comment]
running tests

**Pull request merged**